.circleci/config.base.yml

```diff
@@ -646,6 +646,36 @@ jobs:
           path: ~/repo/packages/amplify-migration-tests/amplify-migration-reports
     working_directory: ~/repo
 
+  amplify_general_config_tests:
+    parameters:
+      os:
+        type: executor
+        default: l_large
+    executor: << parameters.os >>
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
+    steps:
+      - restore_cache:
+          key: amplify-cli-repo-{{ .Branch }}-{{ .Revision }}
+      - restore_cache:
+          key: amplify-cli-yarn-deps-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          key: amplify-pkg-binaries-{{ .Branch }}-{{ .Revision }}
+      - run:
+          name: Run tests using general prpfile
+          command: |
+            source .circleci/local_publish_helpers.sh
+            changeNpmGlobalPath
+            cd packages/amplify-e2e-tests
+            retry yarn run general-config-e2e --no-cache --maxWorkers=3 --forceExit $TEST_SUITE
```

Comment:
why do we need separate job for this?
if we ran this as usual e2e test could we just ignore the default test profile ?

Reply:
because of this, I want a test that not uses .aws folder. Some Cx are sourcing credentials on ec2 instance through IAM role